ClosedXML/Excel/Comments/XLComment.cs

```diff
@@ -151,33 +151,13 @@ public IXLComment SetExtentWidth(Int32 extentWidth)
             return Container;
         }
 
-        #endregion
+        #endregion IXLDrawing
 
         private void Initialize(XLCell cell)
         {
-            Author = cell.Worksheet.Author;
-            Container = this;
-            Anchor = XLDrawingAnchor.MoveAndSizeWithCells;
-            Style = new XLDrawingStyle();
-            Int32 pRow = cell.Address.RowNumber;
-            Double pRowOffset = 0;
-            if (pRow > 1)
-            {
-                pRow--;
-                double prevHeight = cell.Worksheet.Row(pRow).Height;
-                if (prevHeight > 7)
-                    pRowOffset = prevHeight - 7;
-            }
-            Position = new XLDrawingPosition
-                           {
-                               Column = cell.Address.ColumnNumber + 1,
-                               ColumnOffset = 2,
-                               Row = pRow,
-                               RowOffset = pRowOffset
-                           };
+            var style = new XLDrawingStyle();
 
-            ZOrder = cell.Worksheet.ZOrder++;
-            Style
+            style
```

Comment:
```
                style = new XLDrawingStyle();
                style
                .Margins.SetLeft(0.1)
                .Margins.SetRight(0.1)
                .Margins.SetTop(0.05)
                .Protection.SetLocked()
                .Protection.SetLockText();
```

This may be extracted into a static member `DefaultCommentStyle` - it may be useful.
Just a side note.

Reply:
Done.